metpy/calc/tests/test_indices.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2008-2015 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Test the `indices` module."""
+
+from datetime import datetime
+
+from metpy.calc.indices import precipitable_water
```

Comment:
Instead of a direct import like this, you should add `indices` as an import in `calc/__init__.py`. Then here you just:
```python
from metpy.calc import precipitable_water
```